.yamato/config/universal.metafile

```diff
@@ -3,14 +3,22 @@ project:
   folder: UniversalGraphicsTest
 test_platforms:
   - type: Standalone
-    utr_flags:
-      - [OSX_Metal]: --timeout=2400
-    utr_flags_build:
-      #- [Android_OpenGles3, Android_Vulkan]: --timeout=2700
-      #- [Win_DX11, Win_DX12, Win_Vulkan]: --timeout=2000
-      #- [iPhone_Metal]: --timeout=1800
-      - [all]: --timeout=3000
+    timeout:
```

Comment:
same case here as above, might be in more metafiles

Reply:
dunno how this happened, thanks!